scripts/do.sh

```diff
@@ -2,15 +2,28 @@
 # This script is an interface to any of the methods of lib.sh
 # Call this script as "do.sh method_from_lib" to execute any function from that library
 set -eu
+
+command -v devenv >/dev/null 2>&1 || {
+    echo '
+Please install the devenv tool:
+https://github.com/getsentry/devenv#install
+'
+    exit 1
+}
```

Comment:
these should be `if` statements rather than `||` pretending to be an `if`

Reply:
yep, bad habit heh